lib/data_layer.ex

```diff
@@ -2657,9 +2657,11 @@ defmodule AshPostgres.DataLayer do
       case identity do
         %{name: name, where: where} when not is_nil(where) ->
           AshPostgres.DataLayer.Info.identity_where_to_sql(resource, name) ||
-            raise(
-              "Must provide an entry for :#{identity.name} in `postgres.identity_wheres_to_sql` to use it as an upsert_identity"
-            )
+            raise("""
+            Must provide an entry for :#{identity.name} in `postgres.identity_wheres_to_sql` to use it as an upsert_identity.
+
+            See `AshPostgres.DataLayer.Info.identity_wheres_to_sql/1` for an example.
```

Comment:
exception messages w/ backticks like this is a bit annoying because it doesn't resolve to an actual URL. I'd suggest linking directly to hex so users have something to click on.